### ISSUE TITLE
how to change the crop toolbar ok and cancel button

### ISSUE THREAD

**Issue by smallwebbird** [ivpusic/react-native-image-crop-picker#1022]
how to change the crop toolbar ok and cancel button

**Comment by owenzh22** [comment]
cropperChooseText  cropperCancelText

**Comment by smallwebbird** [comment]
but this is only used in ios, should i do something in android

**Comment by ptellesfilho** [comment]
Android doesn't show text for those buttons, but icons instead.

**Comment by smallwebbird** [comment]
yes, now i want to change the icon,can i use a rn component to instead the icon

**Comment by ptellesfilho** [comment]
I believe that currently not, since RN Image Crop Picker uses UCrop underneath which is a native Android dep. This would require some work binding RNICP's wrapper stuff to UCrop, although it does support the option to set a drawable using its Options class (see here https://github.com/Yalantis/uCrop/blob/master/ucrop/src/main/java/com/yalantis/ucrop/UCrop.java#L290)
